bang/magicollections.py

```diff
@@ -790,8 +779,20 @@ def to_fields(self, item, extra_fields=None, exclude_fields=None, order=None, *a
                         'link_text': cameo.name,
                     } for cameo in item.cached_cameos]
                 }))
-            # Add live2d viewer
+            # Add live2d viewer and chibis
             if hasattr(item, 'associated_costume'):
+                chibis_set = item.associated_costume.owned_chibis.all()
```

Comment:
You can have that directly in the queryset line 690:
```python
queryset = models.Card.objects.all().select_related('associated_costume').prefetch_related(Prefetch('associated_costume__owned_chibis', to_attr='costume_chibis'))
```

Then you can just access `item.costume_chibis`.